modules/@apostrophecms/schema/ui/apos/components/AposInputCheckboxes.vue

```diff
@@ -22,7 +22,7 @@
 import AposInputMixin from '../mixins/AposInputMixin';
 
 export default {
-  name: 'AposInputCheckbox',
+  name: 'AposInputCheckboxes',
```

Comment:
This component uses another one called `<AposCheckbox />` but I don't know where this is coming from. If someone can enlight me on this, please.

Reply:
You should see that as a separate file in the `ui` module. Checkboxes are used in other contexts, so we factored it out.

Reply:
Ok, thanks @abea